views/templates/hook/displayBlock.tpl

```diff
@@ -22,22 +22,51 @@
 *  @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
 *  International Registered Trademark & Property of PrestaShop SA
 *}
-<div class="blockreas">
-    {foreach from=$blocks item=$block key=$key}
-        <div class="reass-item" style="{if $block['type_link'] !== $LINK_TYPE_NONE && !empty($block['link'])}cursor:pointer;{/if}"
-                {if ($block['type_link'] !== $LINK_TYPE_NONE) && !empty($block['link'])} onclick="window.open('{$block['link']}')"{/if}>
-            <div class="block-icon">
-                {if $block['icon'] != 'undefined'}
-                    {if $block['icon']}
-                        <img class="svg invisible" src="{$block['icon']}"">
-                    {elseif $block['custom_icon']}
-                        <img src="{$block['custom_icon']}">
+<div class="container">
+    <div class="blockreas">
```

Comment:
If possible, can we improve this class variable ? It's a shortcut for `blockreassurance`.

Reply:
:ok_hand: